packages/modal/src/Modal.stories.tsx

```diff
@@ -72,3 +81,257 @@ export const WithScroll: Story<ModalProps> = (props) => {
     </>
   )
 }
+
+export const LoadingStateInModal: Story<ModalProps> = (props) => {
+  const { state, handleOpen, handleClose } = useModal(props)
+
+  return (
+    <>
+      <Button onClick={handleOpen}>Show loading modal</Button>
+
+      <Modal
+        {...props}
+        titleIcon={<Loader size={'large'} />}
+        open={state}
+        onClose={handleClose}
+      >
+        <Text color='secondary' size='xxs'>
+          Staking 10 ETH. You will receive 10 stETH
+        </Text>
+        <br />
+        <Text color='secondary' size='xxs'>
+          Confirm this transaction in your wallet
+        </Text>
+      </Modal>
+    </>
+  )
+}
+
+LoadingStateInModal.args = {
+  title: 'You are now staking 10 ETH',
+  center: true,
+}
+
+LoadingStateInModal.argTypes = {
+  children: {
+    table: {
+      disable: true,
+    },
+  },
+}
+
+export const SuccessStateInModal: Story<ModalProps> = (props) => {
+  const { state, handleOpen, handleClose } = useModal(props)
+
+  return (
+    <>
+      <Button onClick={handleOpen}>Show success modal</Button>
+
+      <Modal
+        {...props}
+        titleIcon={<Success color={'green'} width={64} height={64} />}
```

Comment:
`color={'green'}` does not work here. The color is used from the svg. It would be great to make it possible to set any color for success and error icons, as it is implemented in other single-color icons

Reply:
fixed